setup.cfg

```diff
@@ -93,6 +93,9 @@ templating =
 plotting =
     matplotlib
     seaborn
+wordcloud =
+     git+https://github.com/kavgan/word_cloud.git
```

Comment:
the package is not available via pypi

There is another package, which seems to have similar features: https://github.com/amueller/word_cloud

Reply:
oh nice! i wonder why I didn't find that one when I did this before, it has almost 9K stars